packages/runtime/container-runtime/src/summaryFormat.ts

```diff
@@ -98,6 +101,14 @@ export function rootHasIsolatedChannels(metadata: IContainerRuntimeMetadata | un
     return !!metadata && !metadata.disableIsolatedChannels;
 }
 
+export function gcEnabled(metadata: IContainerRuntimeMetadata | undefined): boolean | undefined {
```

Comment:
We should probably have an option to toggle this via localStorage for testing in real documents. Similar to this code here:
https://github.com/microsoft/FluidFramework/blob/8b917eac70fb641d9f9f453d7fe76b7b3211d679/packages/loader/driver-utils/src/blobAggregationStorage.ts#L33

Reply:
Do you think it should override the existing flag in the metadata blob?